frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -41,10 +64,22 @@ const TableFilter = (props) => {
     updateSelectedFilter,
   } = props;
 
-  const category =
-    filterData?.length > 0 &&
-    categoryFilterValue &&
-    filterData.filter((item) => item.name === categoryFilterValue)?.[0]?.key;
+
+  tableFilters.forEach(function(entry) {
+    console.log(entry);
+  });
```

Comment:
Probably don't leave this in?

(And, FYI, using a `function` for the callback really looks odd since we use lambdas everywhere else.)